vueui/src/components/header/default.vue

```diff
@@ -28,20 +64,24 @@ export default {
 .uiTitleWrapper {
   margin: -8px -8px 10px;
   position: relative;
-  padding-bottom: 8px;
   float: none;
   background-color: #b9b9b9;
   clear: both;
+  user-select: none;
+  -ms-user-select: none;
+  cursor: default;
+}
+
+.uiTitleWrapper > div:nth-child(1) {
+  height: 32px;
 }
 
 .uiTitleText {
   position: relative;
   display: inline-block;
-  top: 4px;
-  left: 18px;
+  left: 20px;
```

Comment:
I'm not sure I like positioning everything with magic values like this. Is there a particular reason why the elements can't just follow regular flow within the title wrapper, one going after another?